### PR TITLE
cleanup some minor issues

### PR DESCRIPTION
**Pull request by Geosearchef**
just some minor issues/code cleanup

### CODE REVIEW

File: src/map/SCMap.java [hunk 1]
```diff
@@ -80,31 +80,31 @@
...
-	private BufferedImage waterMap;
-	private BufferedImage waterFoamMask;
-	private BufferedImage waterFlatnessMask;
-	private BufferedImage waterDepthBiasMask;
-	private BufferedImage terrainType;
+	private final BufferedImage waterMap;
+	private final BufferedImage waterFoamMask;
+	private final BufferedImage waterFlatnessMask;
+	private final BufferedImage waterDepthBiasMask;
+	private final BufferedImage terrainType;
```
Comment (Geosearchef) [inline src/map/SCMap.java]: can these be final? They are just written into, not changed, aren't they?

Comment (Geosearchef) [inline src/map/SCMap.java]: can these be final? They are just written into, not changed, aren't they?

File: src/generator/PropGenerator.java [hunk 2]
```diff
@@ -52,10 +52,10 @@ private Vector3f placeOnHeightmap(float x, float z) {
...
 	}
 
-	public void randomizeVectorPair(Vector3f v1, Vector3f v2) {
-		placeOnHeightmap(random.nextFloat() * map.getSize(), random.nextFloat() * map.getSize(), v1);
-		placeOnHeightmap(map.getSize() - v1.x, map.getSize() - v1.z, v2);
-	}
+//	public void randomizeVectorPair(Vector3f v1, Vector3f v2) {
+//		placeOnHeightmap(random.nextFloat() * map.getSize(), random.nextFloat() * map.getSize(), v1);
+//		placeOnHeightmap(map.getSize() - v1.x, map.getSize() - v1.z, v2);
+//	}
```
Comment (Geosearchef) [inline src/generator/PropGenerator.java]: not used currently

File: src/export/ScriptExporter.java [hunk 3]
```diff
@@ -5,13 +5,11 @@
...
 public strictfp class ScriptExporter {
-	
-	private static DataOutputStream out;
 
-	public static void exportScript(String folderPath, String mapname, SCMap map) throws IOException {
+    public static void exportScript(String folderPath, String mapname, SCMap map) throws IOException {
 		File file = new File(folderPath + mapname + File.separator + mapname + "_script.lua");
 		file.createNewFile();
-		out = new DataOutputStream(new BufferedOutputStream(new FileOutputStream(file)));
+        DataOutputStream out = new DataOutputStream(new BufferedOutputStream(new FileOutputStream(file)));
```
Comment (Geosearchef) [inline src/export/ScriptExporter.java]: should these stay as static fields so they can be accessed from methods called by `exportScript` without passing references around?

File: src/export/SCMapExporter.java [hunk 4]
```diff
@@ -10,13 +10,13 @@
 
 public strictfp class SCMapExporter {
 
-	public static final byte[] DDS_HEADER_1 = { 68, 68, 83, 32, 124, 0, 0, 0, 7, 16, 0, 0, 0, 1, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
+	private static final byte[] DDS_HEADER_1 = { 68, 68, 83, 32, 124, 0, 0, 0, 7, 16, 0, 0, 0, 1, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
```
Comment (n3roxis) [inline src/export/SCMapExporter.java]: i guess this can be private, yup

File: src/export/SCMapExporter.java [hunk 5]
```diff
@@ -183,14 +183,14 @@ private static void writeBytes(byte[] b) throws IOException {
 	}
 
 	private static void writeShorts(short[] s) throws IOException {
-		for (int i = 0; i < s.length; i++) {
-			writeShort(s[i]);
+		for (short value : s) {
+			writeShort(value);
```
Comment (n3roxis) [inline src/export/SCMapExporter.java]: much cleaner

File: src/map/SCMap.java (hunk 1 above)
Comment (n3roxis) [reply src/map/SCMap.java]: currently they are not changed.

File: src/export/SCMapExporter.java [hunk 6]
```diff
@@ -10,13 +10,13 @@
 
 public strictfp class SCMapExporter {
 
-	public static final byte[] DDS_HEADER_1 = { 68, 68, 83, 32, 124, 0, 0, 0, 7, 16, 0, 0, 0, 1, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
```
Comment (n3roxis) [inline src/export/SCMapExporter.java]: i guess this can be private, yup

**Pull request merged**